centinel/primitives/dnslib.py

```diff
@@ -15,7 +15,7 @@ def get_ips(host, nameserver=None, record="A"):
     return lookup_domain(host, nameservers=nameservers, rtype=record)
 
 
-def lookup_domain(domain, nameservers=[], rtype="A", timeout=10):
+def lookup_domain(domain, nameservers=[], rtype="A", timeout=2):
```

Comment:
> I agree that there are some countries that have RTTs that high, but I don't agree that we should make it so much longer for every country everywhere else. We could do country-specific experiments for cases where RTTs happens to be that high, but for others, I think 2 seconds is pretty reasonable.

I buy that we might want to do country specific RTTs, but only if we didn't have the ability to do concurrency (more on that in a second). Otherwise, setting the largest possible value isn't that big a deal.

> 2 seconds is not inconclusive in most cases. Other cases with higher RTTs can be studied separately. As for concurrency, we don't want to have multiple DNS queries issued at the same time because we are recording pcaps for each URL individually. Doing everything concurrently would make pcaps unclean.

Concurrency shouldn't be an issue for us either at the collection or the analysis stage. If we assume that we are on RPIs, which we previously have, then we can safely collect all traffic to target ports without an risk of collecting user data. For example, we could collect all the data or just collect all of the packets destined for port 53 to get all DNS traffic. Additionally, concurrency won't be an issue at the analysis stage because a) bro is awesome for analyzing pcaps will give us a log of all of the DNS requests and responses in the pcap and b) it is really easy to correlate DNS requests and responses. DNS packets contain a query ID field that *must* match between the query and response and the responses often contain the original question, making identification even easier.

Basically, I don't see a strong argument for not using concurrency. If we have concurrency, then there is no reason to reduce the timeout.